spec/Records.tex

```diff
@@ -508,9 +420,9 @@ \section{The {\em this} Method}
 \section{The {\em these} Method}
 \index{records!iterating}
 
-A \chpl{these} method can be defined for records as well as classes (\rsec{The_these_Method}).  It
-provides an iterator which iterates over the contents of the record in a
-user-defined manner.
+A \chpl{these} method can be defined for records as well as classes
+(\rsec{The_these_Method}).  It provides an iterator which iterates over
```

Comment:
actually I've removed this section since now it just points to Methods.tex and The these Method.